### PR TITLE
cdn and h/w/resize

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: src/http/get-media-000image/index.js [hunk 1]
```diff
@@ -26,11 +26,25 @@ exports.handler = response(async req => {
   if (!originalImage) {
     return reply.text('Not Found', { statusCode: 404 });
   }
+  // set up optimization params
```
Comment (jgallen23) [inline src/http/get-media-000image/index.js:29]: lets only do this logic if resize is set.  I think you also need to call optimiz.resize

Comment (jgallen23) [reply src/http/get-media-000image/index.js:29]: (specifically filename)

File: src/http/get-media-000image/index.js [hunk 2]
```diff
@@ -26,9 +26,25 @@ exports.handler = response(async req => {
...
+  // determine new file name:
+  if (type === 'resize') {
+    const imageBaseTokens = imageName.split('.');
+    const newBaseName = imageBaseTokens.slice(0, imageBaseTokens.length - 1).join('.');
+    imageName = `${newBaseName}_${type}_${width}_${height}.${imageBaseTokens[imageBaseTokens.length - 1]}`;
+  }
   // optimize it:
   try {
-    const imageBuffer = await optimize({ quality: [config.quality, config.quality] }, originalImage.Body);
+    const imageBuffer = await optimize.resize(params, originalImage.Body);
```
Comment (jgallen23) [inline src/http/get-media-000image/index.js:47]: should we be resizing if we don't pass in w/h?  Looks like we lost optimize

Comment (orthagonal) [reply src/http/get-media-000image/index.js:47]: If we don't pass in a w+h, do we want to just store the unmodified image in 'optimized' folder?

Comment (jgallen23) [reply src/http/get-media-000image/index.js:47]: we should optimize the image no matter what

**Pull request merged**